src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final double DEFAULT_DECIMAL_RANK = 10.0;
+    private int radius;
```

Comment:
separate constants and class fields with a new line, for better readability. Do it in each class